internal/Picker/Picker.module.less

```diff
@@ -136,6 +131,11 @@
 			.valueWrapper {
 				font-size: @sand-picker-joined-font-size;
 			}
+			
+			.valueWrapper {
```

Comment:
I'm seeing the same class right below.. 🤔

Reply:
Why this change needed?

Reply:
fixed it.